wiki/Beatmap_Editor/en.md

```diff
@@ -1,13 +1,35 @@
 # Beatmap Editor
 
-The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
+The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). This is where all [beatmapping](/wiki/Beatmapping) takes place, and all content for the game is created. Creating content for osu! is, and always will be, free. So feel free to jump right in and start beatmapping!
 
-- [Compose](Compose): The place where a majority of beatmapping happens. This is where you place circles, sliders, and spinners.
-- [Design](Design): The editor for where Storyboards are made; however, most mappers only use the "Background Image/Video" button here.
-- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hitsounds, and hitsound volume.
-- [Song Setup](Song_Setup): The first place of where all new beatmaps begin. This sets the values regarding the title, artist, version name, difficulty, combo colours, and more.
+The beatmap editor is separated into four main sections which can be seen on the top-left corner of the editor. The categories are:
 
-Other components of the beatmap editor include:
+- Compose
+- Design
+- Timing
+- Song Setup
+
+For Beatmap Editor Shortcuts, see [Shortcut key reference](/wiki/Shortcut_key_reference).
+
+## Compose
+
+Compose is the section where a majority of beatmapping happens. Most mappers spend more time in this category than all of the other categories combined, since this is where gameplay elements like circles, sliders, and spinners are placed. To learn more about this section, see the [compose article](/wiki/Beatmap_Editor/Compose).
```

Comment:
instead of the repeated "to learn more, see this article", you could link the article to the first word. also, since the link goes to something under Beatmap_Editor, you don't need the `/wiki/Beatmap_Editor/` prefix

sometimes we use the "*Main article:*" note at the beginning of sections too, but for something so short its prob not worth doing that